Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -297,10 +350,15 @@ AActor* ABenchmarkGymGameMode::FindPlayerStart_Implementation(AController* Playe
 		return Super::FindPlayerStart_Implementation(Player, IncomingName);
 	}
 
+	if (Player == nullptr) // Work around for load balancing passing nullptr Player
+	{
+		return SpawnPoints[PlayersSpawned % SpawnPoints.Num()];
```

Comment:
Should PlayersSpawned be incremented here? Or is this the instance where it's called twice?

Reply:
Yeah, the load balancing code calls this with Player = nullptr, guess it's just a temporary workaround for now! So it'll return the same point here and below in the second call